{"name": "test_personal_center_workflow", "status": "broken", "statusDetails": {"message": "Exception: 登录过程失败: 打开登录页面失败: 'function' object has no attribute 'get'", "trace": "self = <pages.login_page.LoginPage object at 0x000001AE09272EE0>\n\n    def open_login_page(self):\n        \"\"\"打开登录页面\"\"\"\n        try:\n>           self.driver.get(\"http://192.168.2.176:86/#/case/index\")\nE           AttributeError: 'function' object has no attribute 'get'\n\n..\\pages\\login_page.py:175: AttributeError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <pages.login_page.LoginPage object at 0x000001AE09272EE0>\nusername = 'wxdfg', password = 'wxd341134@'\n\n    def login(self, username=\"wxdfg\", password=\"wxd341134@\"):\n        \"\"\"执行登录操作\"\"\"\n        try:\n            # 打开登录页面\n>           self.open_login_page()\n\n..\\pages\\login_page.py:118: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\login_page.py:179: in open_login_page\n    self.handle_error(f\"打开登录页面失败: {str(e)}\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.login_page.LoginPage object at 0x000001AE09272EE0>\nerror_msg = \"打开登录页面失败: 'function' object has no attribute 'get'\"\ntake_screenshot = True\n\n    def handle_error(self, error_msg, take_screenshot=True):\n        \"\"\"统一的错误处理方法\"\"\"\n        logger.error(error_msg)\n        if take_screenshot:\n            try:\n                allure.attach(\n                    self.driver.get_screenshot_as_png(),\n                    name=\"error_screenshot\",\n                    attachment_type=allure.attachment_type.PNG\n                )\n            except:\n                logger.error(\"无法截图\")\n        # 关闭浏览器\n        try:\n            DriverManager.quit_driver()\n        except Exception as e:\n            logger.error(f\"关闭浏览器失败: {str(e)}\")\n>       raise Exception(error_msg)\nE       Exception: 打开登录页面失败: 'function' object has no attribute 'get'\n\n..\\pages\\login_page.py:46: Exception\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test_usercenter.TestPersonalCenter object at 0x000001AE09272D00>\n\n    @pytest.fixture(autouse=True)\n    def setup_teardown(self):\n        \"\"\"每个测试用例执行前后的设置和清理\"\"\"\n        logger.info(\"开始测试用例执行\")\n    \n        # 使用已有的initialize_driver函数初始化driver，不传入参数\n        # self.driver = initialize_driver()\n    \n        # 执行登录\n        login_page = LoginPage(self.driver)\n>       login_page.login(\"wxdfg\", \"wxd341134@\")\n\nbase_test.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\login_page.py:169: in login\n    self.handle_error(f\"登录过程失败: {str(e)}\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.login_page.LoginPage object at 0x000001AE09272EE0>\nerror_msg = \"登录过程失败: 打开登录页面失败: 'function' object has no attribute 'get'\"\ntake_screenshot = True\n\n    def handle_error(self, error_msg, take_screenshot=True):\n        \"\"\"统一的错误处理方法\"\"\"\n        logger.error(error_msg)\n        if take_screenshot:\n            try:\n                allure.attach(\n                    self.driver.get_screenshot_as_png(),\n                    name=\"error_screenshot\",\n                    attachment_type=allure.attachment_type.PNG\n                )\n            except:\n                logger.error(\"无法截图\")\n        # 关闭浏览器\n        try:\n            DriverManager.quit_driver()\n        except Exception as e:\n            logger.error(f\"关闭浏览器失败: {str(e)}\")\n>       raise Exception(error_msg)\nE       Exception: 登录过程失败: 打开登录页面失败: 'function' object has no attribute 'get'\n\n..\\pages\\login_page.py:46: Exception"}, "description": "测试个人中心完整流程", "attachments": [{"name": "log", "source": "157b9687-ca19-4260-aa99-9ba1b107b930-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "5482274e-3f75-4c6d-b6f6-0844301b29bc-attachment.txt", "type": "text/plain"}], "start": 1744360180802, "stop": 1744360180802, "uuid": "383c23c7-684f-4a86-8025-6019ceba0a8e", "historyId": "bd4ff19a4e8154307673d6417a7a8492", "testCaseId": "bd4ff19a4e8154307673d6417a7a8492", "fullName": "test_usercenter.TestPersonalCenter#test_personal_center_workflow", "labels": [{"name": "feature", "value": "个人中心"}, {"name": "story", "value": "完整测试流程"}, {"name": "suite", "value": "test_usercenter"}, {"name": "subSuite", "value": "TestPersonalCenter"}, {"name": "host", "value": "DESKTOP-3S8JLNL"}, {"name": "thread", "value": "23308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_usercenter"}]}